pkgs/by-name/ti/tiledb/package.nix

```diff
@@ -74,6 +77,8 @@ stdenv.mkDerivation rec {
     cmake
     python3
     doxygen
+    curl.dev
```

Comment:
I'm a bit surprised -- isn't the `dev` output selected by default?

Reply:
Indeed, it is. Removed `.dev`.